File: suivi-scolaire-projet-complet/backend-laravel/app/Models/Absence.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Absence extends Model
{
    protected $fillable = [
        'eleve_id',
        'date_absence',
        'motif',
        'justifiee',
    ];

    protected $casts = [
        'date_absence' => 'date',
        'justifiee'    => 'boolean',
    ];

    public function eleve()
    {
        return $this->belongsTo(Eleve::class);
    }
}

File: suivi-scolaire-projet-complet/mobile-android/app/src/main/java/com/ecole/suiviscolaire/data/model/Absence.kt
package com.ecole.suiviscolaire.data.model

import com.google.gson.annotations.SerializedName

data class Absence(
    @SerializedName("id") val id: Int,
    @SerializedName("date_absence") val dateAbsence: String,
    @SerializedName("motif") val motif: String?,
    @SerializedName("justifiee") val justifiee: Boolean
)
